Source.cpp

```diff
@@ -0,0 +1,105 @@
+#include "Header.h"
+
+Bag::Bag(string description, int severity, float deadline, string status, string assegnee)
+{
+    this->description = description;
+    this->deadline = deadline;
+    this->assegnee = assegnee;
+    this->severity = severity;
+    this->status = status;
+    //cout << "Description - " << description << "\nDifficalty - " << severity << "\nDeadline - " << deadline << "\nStatus - " << status << "\nAssegnee  - " << assegnee;
+}
+Bag::Bag(Bag& B) {
+    this->previous = NULL;
+    this->next = NULL;
+    this->description = B.description;
+    this->severity = B.severity;
+    this->deadline = B.deadline;
+    this->status = B.status;
+    this->assegnee = B.assegnee;
+}
+void Bag::showSomething() const {
+    cout << "Description - " << description << "\nDifficalty - " << severity << "\nDeadline - " << deadline << "\nStatus - " << status << "\nAssegnee  - " << assegnee;
+}
+
+
+void BackLog::insert(Bag* give1) {
+    Bag* give = new Bag(*give1);
```

Comment:
Please, rename give to something more understandable

Reply:
Fixed